int32 num_trigger
float64 delay
---
bool success
string message